logs-to-tsv/README.md

```diff
@@ -2,7 +2,9 @@
 This folder holds two script files. query-logs.js queries AWS Cloudwatch for approximately 200,000 GET requests that look like initial searches from the Discovery Front End. It then writes the api requests to logs-to-tsv/logs-out. write-logs-tsv.js takes the files in that directory and writes them to a tsv.
 
 ## How to use
+`npm i`
 `cd logs-to-tsv`
+`mkdir logs-out`
 `nvm use`
```

Comment:
I think one should need to `cd` before running `nvm use` so that it picks up the `logs-to-tsv/.nvmrc`, right? - and `nvm use` should precede `nvm i`.